src/bootstrap/index.ts

```diff
@@ -0,0 +1,3 @@
+const Thundra = require('@thundra/core');
+
+Thundra.foresightInit();
```

Comment:
Since this is Foresight specific bootstrap, maybe we should put the `index.ts` file under `bootstrap/foresight/ `folder (and so of course move `bootstrap/jest` folder to `bootstrap/foresight/ jest` folder)